### PR TITLE
DEVELOP-1305-2: Updated empty states for extensions: Azure repos

### PR DESCRIPTION
**Pull request by jemmyw**
Add new empty state and menu design

CleanShot 2022-08-11 at 17 29 17@2x

CleanShot 2022-08-11 at 17 29 32@2x

### CODE REVIEW

**Comment by trydionel** [comment]
"Copy ID" button needs to be right-aligned, like you did in https://github.com/aha-develop/github/pull/31
Screen Shot 2022-08-23 at 1 38 02 PM

File: src/lib/extension.ts [hunk 1]
```diff
@@ -1 +1,3 @@
 export const IDENTIFIER = 'aha-develop.azure-devops-repos';
+export const LEARN_MORE_URL = 'https://github.com/aha-develop/azure-devops-repos';
+export const ICON = aha.iconForExtensionIdentifier(IDENTIFIER);
```
Comment (trydionel) [inline src/lib/extension.ts]: This line causes an exception in the endpoint handler.
```
TypeError: aha.iconForExtensionIdentifier is not a function
JavaScript at src/lib/extension.ts:3:25
JavaScript at aha-develop.azure-devops-repos.webhook:247:3
JavaScript at aha-develop.azure-devops-repos.webhook:248:3
JavaScript at Aha.defineExtension (ContributionRegistry.js:50:7)
JavaScript at aha-develop.azure-devops-repos.webhook:1:34
JavaScript at aha-develop.azure-devops-repos.webhook:249:4
```

**Review by trydionel — CHANGES_REQUESTED** [review]

File: src/lib/extension.ts (hunk 1 above)
Comment (jemmyw) [reply src/lib/extension.ts]: @trydionel this should be fixed now

**Comment by jemmyw** [comment]
> "Copy ID" button needs to be right-aligned, like you did in [aha-develop/github#31](https://github.com/aha-develop/github/pull/31) Screen Shot 2022-08-23 at 1 38 02 PM

Done

File: src/lib/extension.ts (hunk 1 above)
Comment (jemmyw) [reply src/lib/extension.ts]: Fixed

**Review by louisfnz — APPROVED** [review]

**Pull request merged**